vMenu/FunctionsController.cs

```diff
@@ -750,54 +766,20 @@ private async Task UpdateLocation()
         /// </summary>
         private void ShowLocation()
         {
-            // Create the default prefix.
-            var prefix = "~s~";
-
-            // If the vehicle node is further away than 1400f, then the player is not near a valid road.
```

Comment:
Since you keep the comment on line 732, it might make sense to reinclude this comment above line 733

Reply:
To be honest, the comment itself doesn't make much sense (what is 1400f?). It's pretty obvious in the ternary anyways what the check is, and why it's needed imo.